src/component/FooterComponent/Copyright/Copyright.jsx

```diff
@@ -0,0 +1,26 @@
+import { Link, Typography } from '@material-ui/core';
+import SocialLinks from '../Links/Social';
+import useStyles from './copyrightStyle';
+
+const Copyright = () => {
+  const classes = useStyles();
+  const { sellyourrecords } = SocialLinks;
```

Comment:
this line is not really necessary. you could just use SocialLinks directly everywhere where this is mentioned.